packages/api/tests/test-bootstrap.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const test = require('ava');
+const bootstrap = require('../lambdas/bootstrap');
+const { randomString } = require('@cumulus/common/test-utils');
+const { Search } = require('../es/search');
+const mappings = require('../models/mappings.json');
+const testMappings = require('./data/testEsMappings.json');
+const mappingsSubset = require('./data/testEsMappingsSubset.json');
+
+let esClient;
+
+test.serial('bootstrap creates index with alias', async (t) => {
+  const indexName = randomString();
+  const testAlias = randomString();
+
+  await bootstrap.bootstrapElasticSearch('fakehost', indexName, testAlias);
+  esClient = await Search.es();
+
```

Comment:
Seems like 14-18 here could be DRY'ed into a beforeEach block or at least a single function that returned indexName, testAlias and esClient.

I am also not sure I understand why each test needs to define `indexName` and `testAlias` locally - in other words why these can't be defined once in this file - but I feel like I've brought that up before so  feel free to ignore this comment if this has already been explained .